crates/glue/tests/fixtures/advanced-recursive-union/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+# ⚠️ READ THIS BEFORE MODIFYING THIS FILE! ⚠️
+#
+# This file is a fixture template. If the file you're looking at is
+# in the fixture-templates/ directory, then you're all set - go ahead
+# and modify it, and it will modify all the fixture tests.
+#
+# If this file is in the fixtures/ directory, on the other hand, then
+# it is gitignored and will be overwritten the next time tests run.
```

Comment:
Hm, according to this comment, this file should be gitignored... 🤨 

Seems like we might not be doing this correctly, although I didn't set up the fixture system and I'm not sure how it's supposed to work. 😅